exampleplugins/Frontend/SwagCustomRiskRule/Bootstrap.php

```diff
@@ -0,0 +1,52 @@
+<?php
+class Shopware_Plugins_Backend_SwagCustomRiskRule_Bootstrap extends Shopware_Components_Plugin_Bootstrap
+{
+    public function install()
+    {
+        $this->subscribeEvent(
+            'Enlight_Controller_Action_PostDispatchSecure_Backend_RiskManagement',
+            'onRiskManagementBackend'
+        );
+
+        $this->subscribeEvent(
+            'Shopware_Modules_Admin_Execute_Risk_Rule_sRiskMyCustomRule',
+            'onMyCustomRule'
+        );
+
+        return true;
+    }
+
+    /**
+     * @param Enlight_Controller_ActionEventArgs $args
+     */
+    public function onRiskManagementBackend(Enlight_Controller_ActionEventArgs $args)
+    {
+        $subject = $args->getSubject();
+        $request = $subject->Request();
+
+        $view = $subject->View();
+        $view->addTemplateDir(__DIR__.'/Views/');
+
+        if ($request->getActionName() === 'load') {
+            $view->extendsTemplate('backend/risk_management/store/risks_store.js');
```

Comment:
Perhaps making the "no namespace collision" more explicit might be helpful: "backend/my_risk_rule/store/risks.js"